rank/scripts/createIndex.ts

```diff
@@ -82,7 +87,7 @@ async function go() {
 
     success('Reindex started successfully')
     info('You can monitor the reindex by running:')
-    code(`yarn checkTask --task ${reindexResp.task.task_id}`)
+    code(`yarn checkTask --task ${reindexResp.task}`)
```

Comment:
I trust that this works, but I can't see how this has changed so that `.task_id` is no longer needed

Reply:
Turns out `task_id` was an incorrect addition the first time round 😅 It didn't work previously, and now it does!